ruma-push-gateway-api/src/notify/send_event_notification/v1.rs

```diff
@@ -112,7 +113,7 @@ pub struct Notification<'a> {
     /// The `content` field from the event, if present. The pusher may omit this
```

Comment:
The `prio` could be just `pub prio: NotificationPriority` with a `serde(default, ...)` to enforce the "If omitted, high is assumed"

Reply:
Yeah that makes sense 👍